spring-cloud-dataflow-deployer-kubernetes/pom.xml

```diff
@@ -1,22 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<artifactId>spring-cloud-dataflow-admin-kubernetes</artifactId>
-	<version>1.0.0.BUILD-SNAPSHOT</version>
+	<artifactId>spring-cloud-dataflow-deployer-kubernetes</artifactId>
 	<packaging>jar</packaging>
-	<name>spring-cloud-dataflow-admin-kubernetes</name>
-	<description>Data Flow Admin for Kubernetes</description>
+	<description>Data Flow deployer for Kubernetes</description>
 	<parent>
 		<groupId>org.springframework.cloud</groupId>
-		<artifactId>spring-cloud-dataflow-parent</artifactId>
+		<artifactId>spring-cloud-dataflow-server-kubernetes-parent</artifactId>
 		<version>1.0.0.BUILD-SNAPSHOT</version>
 		<relativePath />  <!-- lookup parent from repository -->
 	</parent>
 
 	<properties>
-		<start-class>org.springframework.cloud.dataflow.admin.AdminApplication</start-class>
-		<!-- The following version property is inserted in admin.yml -->
-		<spring.cloud.kubernetes.connector.version>1.0.0.BUILD-SNAPSHOT</spring.cloud.kubernetes.connector.version>
+		<!-- The following version property is inserted in server.yml -->
```

Comment:
actually, removing this comment and adding a comment in `spring-cloud-dataflow-server-kubernetes/pom.xml` since that's where the property was moved